package.json

```
@@ -1,14 +1,14 @@
 {
   "name": "swagger-fluent",
-  "version": "5.0.3",
+  "version": "5.0.3-exodus",
```

This way we can iterate our forked version independently:
```suggestion
  "version": "5.0.3-exodus.0",
```